iteration_mapper/README.md

```diff
@@ -15,12 +15,13 @@ Automate the generation of maps in QGIS so that looking at multiple metrics over
 
 #### In QGIS 
 1. Set up a new QGIS project with background layers properly styled. Note these layernames and to add them to the `BACKGROUND_LAYERNAMES` list.
-2. Load a sample layer of data and style it. Save the style to a style file.
+2. Load a sample layer of data and style it. Save the style to a style file. Note that columns must have the same names in any new layer this style is applied to for the style to be properly displayed. 
 3. Set up the print composer.
 4. When adding labels to the print composer, note their ids to add to the `COMPOSER_LABELS` dictionary.
 
    ![](img/composer_items.PNG)
 5. Save the print composer as a template with `Composer > Save as Template...`
+6. Open the template in a new QGIS session to ensure the format is correct
```

Comment:
Can you be more explicit? Open a new print composer using the Template, and explain more how to do that.